docs/release.sh

```diff
@@ -116,17 +84,22 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# replace spaces w %20 so urlencoded
+	local IFS=$'\n'
+	files=( $(find site/ -not -name "*.md*" -not -type f | sed 's/site//g' | sed 's/ /%20/g') )
```

Comment:
Oh man lol I'll fix when I get home

On Thursday, March 19, 2015, Tibor Vass <notifications@github.com> wrote:

> In docs/release.sh
> <https://github.com/docker/docker/pull/11481#discussion_r26812305>:
>
> > @@ -116,17 +84,22 @@ invalidate_cache() {
> >
> >  	aws configure set preview.cloudfront true
> >
> > -	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
> > +	# Get all the files
> > +	# not .md~ files
> > +	# replace spaces w %20 so urlencoded
> > +	local IFS=$'\n'
> > +	files=( $(find site/ -not -name "*.md*" -not -type f | sed 's/site//g' | sed 's/ /%20/g') )
>
> @jfrazelle <https://github.com/jfrazelle> you forgot to remove -not. And
> since we remove whitespaces, we shouldn't need IFS stuff either, but to
> be confirmed :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/11481/files#r26812305>.
>